src/lang/zh-CN.ts

```diff
@@ -2,13 +2,13 @@ import { strTpl } from '../utils';
 import type { Lang } from '.';
 
 export default {
-  exportToOo: '导出为......',
+  exportToOo: '导出为...',
```

Comment:
Please keep Chinese unchanged, this is a language habit.

Reply:
Ok, sorry about that. I will push an update that leaves the Chinese unchanged.